tee-worker/local-setup/launch.py

```diff
@@ -266,30 +273,38 @@ def main(processes, config_path, parachain_type, log_config_path, offset, parach
     parser = argparse.ArgumentParser(
         description="Run a setup consisting of a node and some workers"
     )
-    parser.add_argument("config", type=str, help="Config for the node and workers")
+    parser.add_argument("--config", type=str, help="Config for the node and workers")
     parser.add_argument(
-        "parachain",
+        "--parachain",
         nargs="?",
         default="local-docker",
         type=str,
         help="Config for parachain selection: local-docker / local-binary / remote",
     )
     parser.add_argument(
-        "log_config_path",
+        "--log",
         nargs="?",
         default="./local-setup/worker-log-level-config.toml",
         type=str,
         help="log level config file path"
     )
     parser.add_argument(
-        "offset", nargs="?", default="0", type=int, help="offset for port"
-    )
-    parser.add_argument(
-        "parachain_dir", nargs="?", default="/tmp/parachain_dev", help="Parachain directory for local binary"
+        "--offset", nargs="?", default="0", type=int, help="offset for port"
     )
     args = parser.parse_args()
+    
+    if is_directory_present("/tmp/parachain_dev"): 
+        print("/tmp/parachain_dev already exists, Reassigning the directory")
+        today = datetime.datetime.now()
+        formatted_date = today.strftime('%d-%m-%Y')
+        directory_name = f"parachain_dev_{formatted_date}"
```

Comment:
Why can't we just create new directory with this name, instead of trying with `/tmp/parachain_dev` ?

Reply:
I would say so. Just create `/tmp/parachain_dev_datetime` would be simpler.

Reply:
I believe it's for veterans who are used to look for `/tmp/parachain_dev`

but yea creating new dir each time is fine too

Reply:
Updated ✅